### PR TITLE
refactor: clean up liquidsoap script

### PR DESCRIPTION
**Pull request by smlz**
Couple of clean ups for our player script.

 - saner namings (variables and env variables)
 - some logging
 - code formatting
 - make code a little more robust

### CODE REVIEW

File: klangbecken.liq [hunk 1]
```diff
@@ -4,12 +4,19 @@
...
-#   - KLANGBECKEN_SOCKET_PATH                       #
+#     Default: ./data                               #
+#                                                   #
+#   - KLANGBECKEN_COMMAND                           #
+#     Klangbecken command                           #
+#     Default: python -m klangbecken                #
+#                                                   #
+#   - KLANGBECKEN_PLAYER_SOCKET                     #
 #     Path to socket to listen on                   #
+#     Default: klagbecken.sock                      #
```
Comment (spameier) [inline klangbecken.liq:19]: There's a typo here :wink:

File: klangbecken.liq [hunk 2]
```diff
@@ -65,7 +72,9 @@ def check_next_func(r) =
   filename = request.filename(r)
 
   last_play = float_of_string(metadata["last_play_epoch"])
-  if last_play == 0.0 then
+  if filename == "" then
+    false
```
Comment (spameier) [inline klangbecken.liq:76]: When is the filename empty? Is it because of flac issues you mentioned in #96?

File: klangbecken.liq (hunk 1 above)
Comment (smlz) [reply klangbecken.liq:19]: :face_with_head_bandage:  merci

File: klangbecken.liq (hunk 2 above)
Comment (smlz) [reply klangbecken.liq:76]: This happens from time to time, somewhat at random. I don't know why. It might disappear when updating to Liquidsoap 1.4?!?

Comment (hairmare) [reply klangbecken.liq:76]: Liquidsoap 1.4 is coupled to CentOS 8 if we keep RPMs (and building your own ocaml toolchain on CentOS 7 isn't fun). I'll try again soonish anyway to see if we can get 1.4 available for CentOS 7 as well.

**Review by hairmare — APPROVED** [review]

File: klangbecken.liq (hunk 2 above)
Comment (smlz) [reply klangbecken.liq:76]: For now Liquidsoap 1.3 is working fine. 

And I am all for moving to Liquidsoap 1.4 with an update to CentOS 8.

**Pull request merged**